README.md

```diff
@@ -0,0 +1,24 @@
+# CodeX workflow For Telegram Bot Messages
+
+## Clone   
+
+Clone the repository via command line:
+
+```bash
+$ git clone https://github.com/codex-team/workflow
+```
+
+## Install dependencies
+
+Get required dependencies by the following command.
+```bash
+$ yarn install
+```
+
+## Set Environment Variables & run 
+
+Rename `.env.example` to `.env` and set require env for application.
```

Comment:
we use `.env.sample` in each project. Maybe change accordingly?

Reply:
change to:
Rename `.env.example` to `.env.sample` and set require env for application.